waitress/app/decorators.py

```diff
@@ -11,14 +11,15 @@ def guard(func):
     @wraps(func)
     def decorated_func(viewset, request, pk=None, *args, **kwargs):
         passphrase = request.POST.get('passphrase', None)
+        is_user_permitted = ((not request.user.is_anonymous) and (request.user.is_staff))
         exists = Passphrase.exists(passphrase)
-        if exists.status:
-            request.passphrase = exists.matched_list[0]
+        if exists.status or is_user_permitted:
```

Comment:
This should be broken down into its own if statement. There is already a nested if as the code is before. 

Clean it up perherps?

Reply:
I've removed the other condition, figured I'll do that in a better way than just chaining that in there.